app/code/core/Mage/Review/Block/Customer/List.php

```diff
@@ -130,7 +130,7 @@ public function getProductLink()
     /**
      * Format date in short format
      *
-     * @param $date
+     * @param string $date
```

Comment:
string|Zend_Date|null

Reply:
I'll change and added comment to #784 to recheck after all is merged (Mage_Core eg.)